### ISSUE TITLE
Request for new games: Mosaic and Palisade

### ISSUE THREAD

**Issue by NLZ** [ghewgill/puzzles#9]
Could you please merge in the newest games Mosaic and Palisade from upstream?

**Comment by kohend** [comment]
The creator of the Mosaic game here, I thought this would be simple, but this is using upstream code from 6 years ago and there have been about 50,000 changed lines since, and I'm not an iOS developer.

**Comment by NLZ** [comment]
Yeah, sadly the developer is no longer maintaining this, which is sad because it was a good iOS port.
